src/Propel/Runtime/ActiveQuery/ModelCriteria.php

```diff
@@ -2088,20 +2089,17 @@ protected function getColumnFromName($phpName, $failSilently = true)
             throw new UnknownModelException(sprintf('Unknown model, alias or table "%s"', $prefix));
         }
 
-        if ($tableMap->hasColumnByPhpName($phpName)) {
```

Comment:
this is the actual fix. Note old lines 2102f, where $realColumnName was always set to the name from the table, while $realColumnName was set to the aliased table in line 2094f (now 2097f), but only if column name was a phpName